src/bsp/platform/cc3220sf/xi_bsp_io_net_cc3220sf.c

```diff
@@ -4,7 +4,9 @@
  * it is licensed under the BSD 3-Clause license.
  */
 
-#include <socket.h>
```

Comment:
What happened here?  Did they spilt the function declarations across a series of headers?